resources/lang/zh/admin/pack.php

```diff
@@ -9,8 +9,8 @@
 
 return [
     'notices' => [
-        'pack_updated' => '整合包已经被更新。',
-        'pack_deleted' => '成功删除整合包： ":name" 。',
-        'pack_created' => '一个整合包已被成功创建，现在可以用它来部署服务器了。',
+        'pack_updated' => '已成功更新整合包。',
+        'pack_deleted' => '已成功从系统中删除整合包 “:name”。',
```

Comment:
Is there a reason for the special quotes here rather than standard ones?

Reply:
In Chinese, we don't use standard quotation marks which are halfwidth. We often use fullwidth quotation marks like “”. And of course period (. to 。) etc.

But it's also completely fine to use standard ones or just delete all of them. So, this is more like a personal habit.

Reply:
Interesting, was curious since it wasn't consistent throughout the file, this is fine though.

Reply:
Well try to fix that in next commit :D

BTW, how about that Chinese community thing?